pyquil/operator_estimation.py

```diff
@@ -706,13 +710,20 @@ def group_experiments(experiments: TomographyExperiment,
 @dataclass(frozen=True)
 class ExperimentResult:
     """An expectation and standard deviation for the measurement of one experiment setting
-    in a tomographic experiment.
+    in a tomographic experiment. In the case of readout error calibration, we also include
+    expectation, standard deviation and count for the calibration results, as well as the
+    expectation and standard deviation for the corrected results.
     """
 
     setting: ExperimentSetting
     expectation: Union[float, complex]
     stddev: float
     total_counts: int
+    calibration_expectation: Union[float, complex] = None
+    calibration_stddev: Union[float, complex] = None
+    calibration_counts: int = None
+    corrected_expectation: Union[float, complex] = None
```

Comment:
I highly recommend "flipping" this so `expectation` refers to the corrected expectation and `raw_expectation` optionally includes the uncorrected value. If no correction was requested when running the experiment, `expectation` would be the uncorrected/only value and `raw_expectation` would be `None`. 

Analysis methods should be able to consume the results of `measure_observables` without being privy to which options were selected.

Reply:
Agreed, and updated.